website/docs/basics/blocks-important.md

```diff
@@ -0,0 +1,31 @@
+---
+id: blocks-important
+title: Important
+sidebar_label: Important
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
+
+Before you start developing and changing anything on your block/components, you should keep in mind how we have managed to create this setup and some of the side effects of automatic file loading.
+
+We require all our JS files using the `require.context` and all our SCSS files using the `webpack-import-glob-loader`.
+
+All files are required in the project's` compile-time`, so that means that we can't get any new files after the compile time is over. This approach gives us the ability not to worry if you loaded the file or not. You can focus on your code and leave the messy stuff to the boilerplate.
```

Comment:
```suggestion
All files are required during the project's` compile-time`, so that means that we can't get any new files after the compile time is over. This approach gives us the ability not to worry if you loaded the file or not. You can focus on your code and leave the messy stuff to the boilerplate.
```